src/migration/1704816100142-RemoveUpvoteDownvoteHiddenPostTables.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RemoveUpvoteDownvoteHiddenPostTables1704816100142 implements MigrationInterface {
+    name = 'RemoveUpvoteDownvoteHiddenPostTables1704816100142'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`DROP TABLE "upvote"`);
+      await queryRunner.query(`DROP TABLE "downvote"`);
+      await queryRunner.query(`DROP TABLE "hidden_post"`);
```

Comment:
This is always scary 😬

Reply:
Like imagine, what if there's a really edge case utf-8 bug where `DROP TABLE "upvote"` for some reason gets parsed as `DROP DATABASE` 🤯 
Worst nightmare.

Reply:
What if Chrome forgets to parse JavaScript 😆

Reply:
<img width="281" alt="image" src="https://github.com/dailydotdev/daily-api/assets/9803078/886b7bdc-dee1-45af-b23e-8ff753af220c">